### DISCUSSION TITLE
csuneido mylib

### DISCUSSION THREAD

**Discussion by Claudio08** [apmckinlay/gsuneido#15] [Q&A] [answered]
is possible to load csuneido mylib in gsuneido or is not compatible?

i tried but occur this error:
"PS D:\suneigo> .\gsuneido -load mylib
panic: not a valid dump file [recovered]
        panic: load failed: mylib not a valid dump file"

**Comment by apmckinlay** [comment, marked as answer]
The dump file format did change
so you will not be able to load it.
You could copy the code by opening the dump file in a text editor.
But that will mean copying each record one at a time.
Or you could use LibView > Export from the old cSuneido
and then import into the new gSuneido.
I think the export format is compatible.
